README.md

```diff
@@ -188,10 +188,12 @@ Although these arguments are optional, a (\*) indicates a recommended parameter
 | `debug`             | Enables verbose debug logging.                                                                                                                                                                                                                                                                                                                                                                                                                                           | `false`                        |
 | `defaultBranch`     | The git default branch. The LaunchDarkly UI will default to display code references for this branch.                                                                                                                                                                                                                                                                                                                                                                     | `master`                       |
 | `delimiters` or `D` | Specifies additional delimiters used to match flag keys. Must be a non-control ASCII character. If more than one character is provided in `delimiters`, each character will be treated as a separate delimiter. Will only match flag keys with surrounded by any of the specified delimeters. This option may also be specified multiple times for multiple delimiters. By default, only flags delimited by single-quotes, double-quotes, and backticks will be matched. | `` [" ' `] ``                  |
+| `dryRun`            | If enabled, `ld-find-code-refs` will scan for code references without sending them to LaunchDarkly. May be used in conjunction with `outDir` to output code references data to a csv file instead of sending data to LaunchDarkly.                                                                                                                                                                                                                                       | `false`                        |
+| `outDir`            | Path to an existing directory. If provided, code references will be written to a csv file in the `outDir`                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
You should mention what the filename would be in case they specify a directory already containing many csv files.